extensions/cyclonedx/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,7 @@
+name: "CycloneDX"
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+metadata:
+  keywords:
+  - "cyclonedx"
+  - "cdx"
+  status: "preview"
```

Comment:
Given you added a guide, you could add it there too.

Example here: https://github.com/quarkusio/quarkus/blob/main/extensions/hibernate-validator/runtime/src/main/resources/META-INF/quarkus-extension.yaml#L10